atuin-client/config.toml

```diff
@@ -84,6 +84,9 @@
 ## alt-0 .. alt-9
 # ctrl_n_shortcuts = false
 
+## default history list item format fallback when not appending in cmd
```

Comment:
```suggestion
## default history list format - can also be specified with the --format arg
```

would it also be possible to update this on the docs repo please?

https://github.com/atuinsh/docs